assets/js/libs/ace/ace.js

```diff
@@ -12127,7 +12127,7 @@ top: 0;\
 .ace_scrollbar-h {\
 position: absolute;\
 overflow-x: auto;\
-overflow-y: hidden;\
+overflow-y: auto;\
```

Comment:
Absolutely. My mistake, I didn't realize I was making this change here. I'll override this in the CSS. Well spotted!